tools/tcpaccept.py

```diff
@@ -52,21 +50,20 @@
     // XXX: switch some to u32's when supported
     u64 ts_us;
     u64 pid;
-    u64 ip;
     u64 saddr;
     u64 daddr;
+    u64 ip;
     u64 lport;
     char task[TASK_COMM_LEN];
 };
 BPF_PERF_OUTPUT(ipv4_events);
 
 struct ipv6_data_t {
-    // XXX: update to transfer full ipv6 addrs
     u64 ts_us;
     u64 pid;
+    unsigned __int128 saddr;
```

Comment:
nice. surprised this actually works. I assume the align of the whole struct is still 8. We probably need to add llvm test to make sure it doesn't regress.